vg_chart_generator/README.md

```diff
@@ -0,0 +1,83 @@
+-- Example Usage --
+$ vg-chart-generator --help
+usage: __init__.py [-h] [--columns COLUMNS] [--filtercol FILTERCOL]
+                   [--filterval FILTERVAL] [--values VALUES]
+                   [--groupby GROUPBY] [--chart CHART] [--outfile OUTFILE]
+
+Create Video Game Charts
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --columns COLUMNS     List the available columns
+  --filtercol FILTERCOL
+                        The column to use for filtering
+  --filterval FILTERVAL
+                        The value of filtercol to use for filtering
+  --values VALUES       List the values for provided filter column
+  --groupby GROUPBY     The column to use for grouping the filtered data
+  --chart CHART         Save the filtered and grouped data as a chart
+  --outfile OUTFILE     The output file name for the generated char
+
+
+-- List Columns --
+
+$ vg-chart-generator --columns
+Name
+Platform
+Year_of_Release
+Genre
+Publisher
+NA_Sales
+EU_Sales
+JP_Sales
+Other_Sales
+Global_Sales
+Critic_Score
+Critic_Count
+User_Score
+User_Count
+Developer
+Rating
+
+
+-- List Values for a Filter Column --
+$ vg-chart-generator --filtercol Publisher --values
+10TACLE Studios
+1C Company
+20th Century Fox Video Games
+2D Boy
+3DO
+49Games
+505 Games
+5pb
+7G//AMES
+989 Sports
+989 Studios
+AQ Interactive
+...
+
+-- View Statistics with a specified filter and groupby --
+$ vg-chart-generator --filtercol Publisher --filterval "Square Enix" --groupby Platform
+Platform
+GBA      2
+WiiU     3
+PSV      9
+Wii      9
+XOne    11
+3DS     14
+PC      20
+PSP     20
+PS4     21
+X360    25
+PS2     26
+PS3     29
+DS      45
+
+-- Print Chart with autogenerated file name --
+$ vg-chart-generator --filtercol Publisher --filterval "Square Enix" --groupby Platform --chart
+
+Generated file: Square Enix_Platform.png
+
+-- Override generated file name --
+
+$ vg-chart-generator --filtercol Publisher --filterval "Square Enix" --groupby Platform --chart --outfile ../../images/square_enix_games_by_platform.png
```

Comment:
And this one I'm pretty sure is the readme file to help others trying to use this package understand how to use it.